sdk/go/common/resource/plugin/analyzer_plugin.go

```diff
@@ -601,7 +601,7 @@ func convertConfigSchema(schema *pulumirpc.PolicyConfigSchema) *AnalyzerPolicyCo
 	props := make(map[string]JSONSchema)
 	for k, v := range unmarshalMap(schema.GetProperties()) {
 		s := v.(map[string]interface{})
-		props[k] = JSONSchema(s)
+		props[k] = s
```

Comment:
Seems like we should just change this to one line:

```go
		props[k] = s.(JSONSchema)
```

Reply:
Thanks for the advice, it looks like my mind is out.